risc0/binfmt/src/image.rs

```diff
@@ -71,10 +71,19 @@ pub struct PageTableInfo {
 
 impl From<PersistentPageTableInfo> for PageTableInfo {
     fn from(value: PersistentPageTableInfo) -> Self {
-        PageTableInfo::new(value.page_table_addr, value.page_size)
+        PageTableInfo::new(value.page_table_addr, value.page_size).expect("Invalid page table")
     }
 }
 
+// TODO: Decide between TryFrom and From
```

Comment:
@Cardosaum were there any disadvantages to `TryFrom`? I think you mentioned it to me but I forgot...

Reply:
`expect` causes a panic, so I don't think we can use `From` here.

Reply:
@SchmErik , the problem I was having was because the struct definition was specifying `serde(from = "PersistentPageTableInfo")` instead of `serde(try_from = "PersistentPageTableInfo")`, a simple detail I was overlooking.